third_party/netcdf-c.BUILD

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2022 Swift Navigation Inc.
+# Contact: Swift Navigation <dev@swift-nav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+"""
+netcdf-c@4.9.0
+"""
+
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+filegroup(
+    name = "allsrcs",
+    srcs = glob(["**"]),
+)
+
+cmake(
+    name = "netcdf-c",
+    cache_entries = {
+        "BUILD_SHARED_LIBS": "OFF",
+        "CMAKE_BUILD_TYPE": "Release",
```

Comment:
Did you need this to build? We might want to check if `rules_foriegn_cc` handles the build type for us (i.e. if we use `bazel build -c dbg` it will automatically pass the corresponding option `-DCMAKE_BUILT_TYPE=Debug` to cmake.

Reply:
nope, looks like it builds fine without it